database/insertData_SAFETY.js

```diff
@@ -0,0 +1,2340 @@
+const fetch = require('node-fetch')
```

Comment:
is this a back up file ?

Reply:
Yes, this was the original file. Was just keeping it around in case I needed to compare the copied queries with the original. Was planning on deleting before merge.